src/devices/wec2103.c

```diff
@@ -39,33 +39,73 @@ Example datagram:
 - Example: 82.4F->824->1724->0x6bc
 */
 
+static uint8_t almost_crc4(uint8_t const message[], unsigned nBytes, uint8_t polynomial, uint8_t init)
```

Comment:
The doc comment now needs to start with
`/** @fn int wec2103_decode(r_device *decoder, bitbuffer_t *bitbuffer)`
as not to attach to this helper function.

Reply:
Done. There's no precedent for this in the code, though. See e.g. blueline.c.

Reply:
Some of the doc-comments are broken, yes. PR if you have the time and like to fix some :)